cmake/modules/StandaloneCrt.cmake

```diff
@@ -40,129 +40,143 @@ if(MSVC)
 
 else()
 
```

Comment:
NOTE: change diff settings to hide whitespace (if not set) for this diff to make more sense since I needed to change indentation.